test/parallel/test-stream-err-multiple-callback-construction.js

```diff
@@ -0,0 +1,23 @@
+const stream = require('stream');
+const assert = require('assert');
+
+class TestWritable extends stream.Writable {
+  _write(chunk, encoding, callback) {
+    callback();
+  }
+
+  _final(callback) {
+    process.nextTick(callback);
+    process.nextTick(callback);
+  }
+}
+
+const writable = new TestWritable();
+
+writable.on('finish', () => {});
```

Comment:
```suggestion
writable.on('finish', common.mustNotCall());
```

Reply:
I don't agree with this. The finished callback will be called at least once (it will be called twice), the error only occurs when it's called for a second time. Introducing this change makes the test fail.

Reply:
Use `common.mustCall()` instead of `common.mustNotCall()` then.

Reply:
The `finish` callback should be called only once. I didn't word my original message the best (as the second invocation should throw the error we're intending to catch.) I've changed this to use `mustCall`.